### ISSUE TITLE
New vegetation departure fields to parse riparian loss and invasive encroachment

### ISSUE THREAD

**Issue by mhallerud** [Riverscapes/RCAT#17]
@wally-mac, Jenna Walsh (from the USFS PIBO team) and I have been investigating RCAT outputs for the PIBO project and have realized a few improvements that could be made.  One improvement that we think would be beneficial is calculating two forms of riparian vegetation departure as follows:

1. Riparian loss = (proportion of area with existing riparian vegetation [including nonnatives]) / (proportion of area with historic riparian vegetation)
2. Invasive encroachment = (proportion of area with existing **native** riparian vegetation) / (proportion of area with historic **native** riparian vegetation)

We believe that this change will give a better understanding of what is driving vegetation changes on the ground.

**Comment by mhallerud** [comment]
These new fields will be `RIPAR_DEP` for point 1 described above and `NATIV_DEP` for part 2. `NATIV_DEP` is equivalent to the previously calculated `DEP_RATIO` and represents the change in native riparian vegetation.

**Comment by mhallerud** [comment]
This problem has been fully addressed in [RCAT Version 2.0](http://rcat.riverscapes.xyz/Documentation/Version_2.0/).
